src/db/sysdb.h

```diff
@@ -215,7 +215,7 @@
 #define SYSDB_GRSID_FILTER "(&("SYSDB_GC")("SYSDB_SID_STR"=%s))"
 #define SYSDB_GRENT_FILTER "("SYSDB_GC")"
 #define SYSDB_GRNAM_MPG_FILTER "(&("SYSDB_MPGC")(|("SYSDB_NAME_ALIAS"=%s)("SYSDB_NAME_ALIAS"=%s)("SYSDB_NAME"=%s)))"
-#define SYSDB_GRGID_MPG_FILTER "(&("SYSDB_MPGC")("SYSDB_GIDNUM"=%lu))"
+#define SYSDB_GRGID_MPG_FILTER "(|(&("SYSDB_GC")("SYSDB_GIDNUM"=%lu))(&("SYSDB_UC")("SYSDB_GIDNUM"=%lu)("SYSDB_UIDNUM"=%lu)))"
```

Comment:
would it be more readable if we use positioning of arguments?

    #define SYSDB_GRGID_MPG_FILTER "(|(&("SYSDB_GC")("SYSDB_GIDNUM"=%1$lu))(&("SYSDB_UC")("SYSDB_GIDNUM"=%1$lu)("SYSDB_UIDNUM"=%1$lu)))"

It is just a tought, I see that it is not used in other filters too.

Reply:
This is a cherry-pick from 2.x branch. So if we want to change this, we need to start with 2.x branch.

Reply:
Thank you for pointing this out. Then it is not worth the effort. ACK